### DISCUSSION TITLE
Icon description repeated for rain_sleet and snow_sleet

### DISCUSSION THREAD

**Discussion by sdennstaedt** [weather-gov/api#395] [Q&A]
Noticed that here:   https://api.weather.gov/icons

it says in the JSON, 

        "rain_sleet": {
            "description": "Rain/sleet"
        },
        "snow_sleet": {
            "description": "Rain/sleet"
        },

Is that a typo that they both say Rain/sleet under the description for both rain_sleet & snow_sleet?

**Comment by StephenClouse** [comment]
Yep. It's fixed in our next release.
